### PR TITLE
[SDKNEW-1654] Feature/new block page

### PR DESCRIPTION
**Pull request by ori-gold-px**
https://perimeterx.atlassian.net/browse/SDKNEW-1654

### CODE REVIEW

File: src/main/java/com/perimeterx/api/blockhandler/templates/TemplateFactory.java [hunk 1]
```diff
@@ -41,28 +41,29 @@ public static String getTemplate(String template, Map<String, String> props) thr
         Map<String, String> props = new HashMap<>();
 
         props.put("appId", pxConfig.getAppId());
-        props.put("refId", pxContext.getUuid());
         props.put("vid", pxContext.getVid());
         props.put("uuid", pxContext.getUuid());
         props.put("customLogo", pxConfig.getCustomLogo());
+        props.put("logoVisibility", pxConfig.getCustomLogo() == null ? "hidden" : "visible");
```
Comment (AsafAklerPX) [inline src/main/java/com/perimeterx/api/blockhandler/templates/TemplateFactory.java:47]: No need for logo visibility any more..

File: src/main/resources/com/perimeterx/api/blockhandler/templates/block_template.mustache [hunk 2]
```diff
@@ -115,7 +115,7 @@
                 reference ID below.
             </p>
             <p>
-                Reference ID: #{{refId}}
+                Reference ID: #{{uuid}}
```
Comment (AsafAklerPX) [inline src/main/resources/com/perimeterx/api/blockhandler/templates/block_template.mustache:118]: we removed the refId don't we? why adding uuid here?

File: src/test/resources/block.mustache [hunk 3]
```diff
@@ -115,7 +114,7 @@
                 reference ID below.
             </p>
             <p>
-                Reference ID: #{{refId}}
+                Reference ID: #{{uuid}}
```
Comment (AsafAklerPX) [inline src/test/resources/block.mustache:117]: reference id?

**Review by AsafAklerPX — CHANGES_REQUESTED** [review]
refrence id? also

File: src/main/java/com/perimeterx/api/blockhandler/templates/TemplateFactory.java (hunk 1 above)
Comment (ori-gold-px) [reply src/main/java/com/perimeterx/api/blockhandler/templates/TemplateFactory.java:47]: Keeping it for hard block page which requires it

File: src/main/resources/com/perimeterx/api/blockhandler/templates/block_template.mustache (hunk 2 above)
Comment (ori-gold-px) [reply src/main/resources/com/perimeterx/api/blockhandler/templates/block_template.mustache:118]: This is the hard block page -- can't use refId so using uuid instead (same value)

File: src/test/resources/block.mustache (hunk 3 above)
Comment (ori-gold-px) [reply src/test/resources/block.mustache:117]: They're the same

**Review by AsafAklerPX — APPROVED** [review]

**Pull request merged**